test/isolate_test.dart

```diff
@@ -430,6 +430,51 @@ void main() {
     });
   });
 
+  group("resume(overAsyncSuspension)", () {
+    var isolate;
+    var stdout;
+    setUp(() async {
+      client = await runAndConnect(topLevel: r"""
+        inner() {
+          print("in inner");
+          return new Future.delayed(const Duration(milliseconds: 1));
+        }
+
+        outer() async {
+          debugger();
+          await inner(); // line 9
+          print("after inner"); // line 10
+        }
+      """, main: r"""
```

Comment:
trailing comma style might make this code more readable.
E.g. something like
```dart
 await runAndConnect(
    topLevel: r"""
        inner() {
          print("in inner");
          return new Future.delayed(const Duration(milliseconds: 1));
        }

        outer() async {
          debugger();
          await inner(); // line 9
          print("after inner"); // line 10
        }
      """,
    main: r"""
        await outer();
        print("after outer");
      """,
  )   


trailing comma style might make this code more readable.

Attach files by dragging & dropping, , or pasting from the clipboard.  Styling with Markdown is supported

Reply:
The only difference it makes is that the closing paren goes on the next line. This is formatted the same as the rest of the file though, so I'm not sure if to commit that.